src/groupByKey.ts

```diff
@@ -2,6 +2,15 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof {}): GroupsMap<T> {
```

Comment:
Keys of plain object? Doesn't look right
```suggestion
export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
```